api/util/logging.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Builds a middleware that filters the body logs
+ * @param w Whitelist (an array of fields)
+ * @param b Blacklist (an array of fields)
+ * @returns {Function}
+ */
+exports.bodyLogFilter = ({w, b}) => {
+  return (req, res, next) => {
+    req._routeWhitelists.body = w || req._routeWhitelists.body;
+    req._routeBlacklists.body = b || [];
+    next();
+  };
+};
+
+
+exports.logError = (logger, err) => {
+  const err_info = winston.exceptions.getAllInfo(err);
```

Comment:
@waliens `winston` not declared?

Reply:
Yep, don't know why it didn't show up while I was testing the PR.